content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_dialog/.content.xml

```diff
@@ -109,6 +109,7 @@
                                                 autoStart="{Boolean}false"
                                                 class="cq-droptarget"
                                                 fieldLabel="Background Image"
+                                                fieldDescription="Background image for the container, please use images that will provide adequate contrast"
```

Comment:
I'd add "... with the text (accessibility)" at the end of the description.

Reply:
Looks good to me.